content/shell/browser/shell_content_browser_client.cc

```diff
@@ -300,13 +300,15 @@ std::unique_ptr<PrefService> CreateLocalState() {
 }  // namespace
 
 std::string GetShellUserAgent() {
```

Comment:
Can you #ifdef get this with `#if COBALT` so that the default build does not change behavior ?